docs/overview.rst

```diff
@@ -430,6 +430,11 @@ TMT_PLAN_DATA
     back from the guest and available for inspection after the
     plan is completed.
 
+TMT_PLAN_ENVIRONMENT_FILE
+    Path to the file containing environment variables that should
+    be sourced after prepare and execute steps. These variables will
+    then be accessible for all subsequent steps.
```

Comment:
Perhaps we should explicitly mention the expected format of the file? And include an example to make it perfectly clear?

Reply:
For some recent variables we've also included a `versionadded` mention. @lukaszachy, @happz, would it make sense here as well?

Reply:
Added in 9ecf336.